server/database/config/connection.ts

```diff
@@ -31,6 +30,4 @@ const sequelize = new Sequelize(dbUrl, {
   logging: false,
 });
 
-sequelize.sync();
```

Comment:
We need this for both production and development, But maybe we can wrap it in a condition!
MAYBE in the build script, you can add environment variable like
DB_BUILD=true
and here you can sync sequelize when DB_BUILD equals false